### DISCUSSION TITLE
cQube Student Assessment Insights - Sample

### DISCUSSION THREAD

**Discussion by Divx-P** [Sunbird-cQube/community#869] [Q&A] [answered]
_HI, I''m trying to understand if cqube could be used for creating student profiles & learning progress across multiple areas. Eg: Capturing qualitiative feedback from multiple teachers,  capturing pscyho educational data , actual formative/ summative assessments across subjects etc...      Would it be possible to get a sample of what data format needs to be ingested and insights feasible? 

The approach of being able to ingest data (eg with OCR), visualize progress and focus areas for multiple criteria for student/ class/ district is  key. 

P.S: Im not a dev-ops person and trying to understand capabilities for a specific use case

**Comment by PavitraIR** [comment]
@Divx-P 

Please provide us with an example data format according to your requirements so that we can determine if the specific use case data will be supported or not.
For further details, please refer to the provided document. https://cqube.sunbird.org/use-cqube/getting-started-1

**Reply by Divx-P** [reply]
HI Pavitra.  I've added some sample data here. 

I did go through the data in the link you provided but it doesnt have the granularity for Student Progression within each subject.
 Instead of only the overall academic year grade by subject, is it possible to add more granularity within each subject/assessment on progress. 
 Ability to add assessment specific parameters for each 'subject"

[Sample Student Profile Progression Data.xlsx](https://github.com/Sunbird-cQube/community/files/14860922/Sample.Student.Profile.Progression.Data.xlsx)

**Reply by Divx-P** [reply]
Primarily looking to see if there is a way to manage Longitudinal Student Information Management

**Reply by PavitraIR** [reply, marked as answer]
Hi @Divx-P 

Thanks for reaching out. We do not work student-specific information in cqube, as cqube does not store such data. If you wish to proceed, you will need to create a dimension grammar that aligns with the data structure of dimension grammars. Then you can ingest the data accordingly.

**Reply by PavitraIR** [reply]
Your data does not include a metric column, which is necessary for data interpretation. For instance, here is a sample metric data example with three columns: "enrolled," "meal_served," and "total_schools." These metrics provide specific measures that are vital for data analysis and understanding.

https://docs.google.com/spreadsheets/d/1AILPkUeoPtAm-Wx8BU5dHmVqcCcU2W34QNggpY8sDAs/edit#gid=0

**Reply by Divx-P** [reply]
Thank you Pavithra. Is there any other module of sunbird which could be better suited for student information management and reporting ? 

Did  see the metrics didn’t have anything for student specific info but we’re already setup to handle school data. That’s why thought I’ll check if we could make it a bit more granular.

**Reply by PavitraIR** [reply]
Hi @Divx-P 

The only available module in Sunbird within CQube is what we can utilize. To proceed, you'll need to create Key Performance Indicators (KPIs), grammar files and then you can ingest the new program. Please refer to the provided documents for more information.

https://cqube.sunbird.org/use-cqube/getting-started-1/schema

https://cqube.sunbird.org/use-cqube/visualizing-the-data/enhance-customize-cqube/manual-and-automated-testing

**Comment by Divx-P** [comment]
Thanks Pavitra. Will check it out.
